addon/components/frost-toggle.js

```diff
@@ -130,8 +163,8 @@ export default Component.extend(PropTypeMixin, FrostEventsProxy, {
       }
     },
     /* eslint-enable complexity */
-    noop () {
-      // we have this method to avoid action undefined error
-    }
+
+    /** A no-op action to avoid an action undefined error */
```

Comment:
If a jsdoc comment is necessary, I'd prefer the single line `/** */` format so that I don't have to look across too many meaningless lines when scanning a file